docs/index.md

```diff
@@ -1,69 +0,0 @@
----
```

Comment:
I think this should come back right, as right now you have deleted the main page of the Algorithmkader